includes/models/model.llms.user.certificate.php

```diff
@@ -111,4 +111,49 @@ public function get_user_postmeta() {
 		);
 	}
 
+	/**
+	 * Can user manage and make some actions on the certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int|null $user_id Optional. WP User ID (will use get_current_user_id() if none supplied). Default `null`.
+	 * @return bool
+	 */
+	public function can_user_manage( $user_id = null ) {
+
+		$user_id = $user_id ?? get_current_user_id();
+		$result  = ( $user_id == $this->get_user_id() || llms_can_user_bypass_restrictions( $user_id ) );
+
+		return apply_filters( 'llms_certificate_can_user_manage', $result, $this );
+
+	}
+
+	/**
+	 * Can user view the certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int|null $user_id Optional. WP User ID (will use get_current_user_id() if none supplied). Default `null`.
+	 * @return bool
+	 */
+	public function can_user_view( $user_id = null ) {
+
+		$user_id = $user_id ?? get_current_user_id();
```

Comment:
Use ternary in favor of null coalesce operator.

Why?

While I have been called a dinosaur for adhering to rules like this that exist to support unsupported versions of PHP, I prefer to stay out of fights and arguments about progressive PHP in favor of focusing on what I'm personally better at.

This is to say that currently, the WordPress core does not allow usage of the null coalesce operating so neither do we.

Source if interested: https://make.wordpress.org/core/2020/03/20/updating-the-coding-standards-for-modern-php/ see the "Operators" section

```suggestion
		$user_id = $user_id ? $user_id : get_current_user_id();
```

Reply:
But they said:
> The spaceship <=>, null coalesce ?? and null coalesce equals ??= operators can not be used in WordPress Core **until the minimum PHP version has been raised to PHP 7.0**

and i noticed the min php version you support is 7.2, and that's why i used this operator so why not ?

anyway i will make as you want, it's not a big deal

Reply:
We adhere (mostly) to the WP core coding standards. The more exceptions we make the more complicated it becomes to maintain our own set of coding standards. We are a small team (unlike the WP core). I don't have the capacity to keep up with all the newest and most "golfy" operators and standards. So I simply adopt what the WP core does (even though we have different minimum version requirements).

Thanks for updating.